workflows/funcscan.nf

```diff
@@ -362,8 +362,6 @@ workflow FUNCSCAN {
         )
     )
 
-    ch_multiqc_files = ch_multiqc_files.mix( ANNOTATION.out.multiqc_files.collect{it[1]}.ifEmpty([]) )
```

Comment:
Why is this being deleted? PROKKA and BAKTA have MultiQC modules

Reply:
Ahhh I see! 128 is wrong, that should be removed (it should have the ifEmpty to make sure MQC will still execute for software versions by satisfying having _something_  in the channel). The 128 line must be a copy paste mistake from the versions thing

Reply:
Okay thank you! Will fix